src/qml/QFieldCamera.qml

```diff
@@ -156,38 +212,176 @@ Popup {
       focus: visible
     }
 
-    QfToolButton {
-      id: okButton
+    Rectangle {
+      x: cameraItem.isPortraitMode ? 0 : parent.width - 100
+      y: cameraItem.isPortraitMode ? parent.height - 100 : 0
+      width: cameraItem.isPortraitMode ? parent.width : 100
+      height: cameraItem.isPortraitMode ? 100 : parent.height
+
+      color: Qt.hsla(Theme.darkGray.hslHue, Theme.darkGray.hslSaturation, Theme.darkGray.hslLightness, 0.3)
+
+      Rectangle {
+        id: captureRing
+        anchors.centerIn: parent
+        width: 64
+        height: 64
+        radius: 32
+        color: Qt.hsla(Theme.darkGray.hslHue, Theme.darkGray.hslSaturation, Theme.darkGray.hslLightness, 0.3)
+        border.color: cameraItem.state == "VideoCapture" && camera.videoRecorder.recorderState != CameraRecorder.StoppedState
+                      ? "red" : "white"
```

Comment:
```suggestion
                      ? "red"
                      : "white"
```

Lets have it consistent, either 1 when condition is short or 3 lines when too long.

Reply:
Agreed.